modules/machineset-enabling-efa-options.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-enabling-efa-options_{context}"]
+= Enabling MPI workloads that use an AWS Elastic Fabric Adapter
+
+After configuring a machine set to support the use of an AWS Elastic Fabric Adapter (EFA), you must install additional software to run MPI workloads. 
+
+For more information about using Kubeflow and the MPI Operator in {product-title} and an example, see link:https://cloud.redhat.com/blog/how-to-use-kubeflow-and-the-mpi-operator-on-openshift[How to use Kubeflow and the MPI Operator on OpenShift].
```

Comment:
Why are we using a blog; as the source of 'how to install' and get setup to use EFA capabilities'?

Reply:
Why are we cloning this (from: https://github.com/kubeflow/mpi-operator); vs instilling it from Operator Hub?